website/docs/r/is_lb.html.markdown

```diff
@@ -45,7 +45,10 @@ Review the argument references that you can specify for your resource.
 
 - `logging`- (Optional, Bool) Enable or disable datapath logging for the load balancer. This is applicable only for application load balancer. Supported values are **true** or **false**. Default value is **false**.
 - `name` - (Required, String) The name of the VPC load balancer.
-- `profile` - (Required, Forces new resource, String) The profile to use for this load balancer. Supported value is `network-fixed`.
+- `profile` - (Optional, Forces new resource, String) The profile to use for this load balancer. Supported value is `network-fixed`.
+  
+  **Note** `profile` should be set to `network-fixed` to create a network load balancer. To create an application load balancer do not set value to `profile`. Application load balancer is created by default when `profile` is not set.
+
```

Comment:
- `profile` - (Optional, Forces new resource, String) For a Network Load Balancer, this attribute is required and should be set to `network-fixed`. For Application Load Balancer, `profile` is not a required attribute.